src/scripts/validation.js

```diff
@@ -56,13 +57,13 @@ const resetValidation = (formEl, config) => {
   inputList.forEach((inputElement) => {
     hideInputError(formEl, inputElement, config);
   });
-  disableButton(buttonElement, config);
+  toggleButtonState(inputList, buttonElement, config);
 };
 
 const setEventListeners = (formEl, config) => {
   const inputList = Array.from(formEl.querySelectorAll(config.inputSelector));
   const buttonElement = formEl.querySelector(config.submitButtonSelector);
-  toggleButtonState(inputList, buttonElement, config);
+  // toggleButtonState(inputList, buttonElement, config);
```

Comment:
`  // toggleButtonState(inputList, buttonElement, config);
`

Please, uncomment it. You need it to initially disable the button